.run/MainApp.run.xml

```diff
@@ -0,0 +1,9 @@
+<component name="ProjectRunConfigurationManager">
+  <configuration default="false" name="MainApp" type="Application" factoryName="Application">
+    <option name="MAIN_CLASS_NAME" value="core.basesyntax.MainApp" />
+    <module name="oop-example" />
+    <method v="2">
+      <option name="Make" enabled="true" />
+    </method>
+  </configuration>
+</component>
```

Comment:
what is this?

Reply:
i don't know, i hadn't changed this file. Should i delete this?

Reply:
delete this file from PR here, on GitHub